internal/apps/poll/restrict.go

```diff
@@ -36,24 +36,24 @@ func RestrictPoll(r restricter.HasPermer, canSee, canManage string, restrictedFi
 		}
 		poll["user_has_voted"] = hasVoted
 
-		// get the users `vote_delegated_from_users_id`
+		// Get the users `vote_delegated_from_users_id`.
 		var user struct {
 			VoteDelegationIds []int `json:"vote_delegated_from_users_id"`
 		}
 		if err := r.Get("users/user", uid, &user); err != nil {
 			return nil, fmt.Errorf("unmarshal user: %w", err)
 		}
-		// Calc the intersection of voteDelegationIds and votedID
-		hasVotedForDelegationIds := []int{}
+		// Calc the intersection of voteDelegationIds and votedID.
+		ids := []int{}
```

Comment:
I don't know if there is a standard way to initialize a varaible. In this case, there are tree possible ways:
```
ids := []int{}
```

```
ids := make([]int, 0)
```

```
var ids []int
// later
ids = append(ids, id)
```

I try to always use the same ways in this repo. If I know the length of a slice or map, I initialize it with `make()`. If I don't know the length of a slice, I use `var NAME []TYPE`.

There are some exceptions of this rule. For example if a slice is encoded to json, then it makes a different, if is nil or empty.

I use the form []TYPE{....} when the variable is initialized and not changed afterwards.

In this case, please use

```
var ids []int
```

Maybe you can ask at intevation what they say about this topic.